src/celeritas/io/LivermorePEReader.cc

```diff
@@ -117,7 +117,7 @@ LivermorePEReader::operator()(AtomicNumber atomic_number) const
                 infile >> result.xs_lo.x[i] >> result.xs_lo.y[i];
             }
         }
-        else if (atomic_number <= AtomicNumber{2})
+        else if (atomic_number > AtomicNumber{2})
```

Comment:
God damn it... maybe add a pull request to fix this bug I introduced (#1157) and if we really want to be sure then add check using a scoped logger.

Reply:
No prob, I'll split this fix out too.